packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -100,20 +89,6 @@ export class W3mAccountDefaultWidget extends LitElement {
 
       <wui-flex flexDirection="column" gap="xs" .padding=${['0', 's', 's', 's'] as const}>
         ${this.authCardTemplate()} <w3m-account-auth-button></w3m-account-auth-button>
-
-        <wui-list-item
-          .variant=${networkImage ? 'image' : 'icon'}
-          iconVariant="overlay"
-          icon="networkPlaceholder"
-          imageSrc=${ifDefined(networkImage)}
-          ?chevron=${this.isAllowedNetworkSwitch()}
-          @click=${this.onNetworks.bind(this)}
-          data-testid="w3m-account-select-network"
-        >
-          <wui-text variant="paragraph-500" color="fg-100">
-            ${this.network?.name ?? 'Unknown'}
```

Comment:
A design problem; previously when we switch to unknown network, we were typing `Unknown` to the button. But now the new network select item doesn't have text, when users open the network select page, they won't see any of the networks selected, but we should show them at least the chain id.